build.gradle

```diff
@@ -144,6 +142,7 @@ publishing {
 }
 
 spotbugs {
+    toolVersion = '4.8.0'
```

Comment:
I think we'd be better with https://github.com/IBM/cloudant-spring/pull/64 moving the plugin version to `5.2.1`  (the plugin includes `4.8.0` since `5.1.5`).
I think I'll override the merge there so that you can rebase on it, otherwise we have a chicken/egg problem to get a green build.